generator/filesystem.cpp

```diff
@@ -59,11 +79,18 @@ static std::error_code create_directory(const llvm::Twine& path)
     using namespace llvm;
```

Comment:
I think the create_directories function should actually simply call (or be replaced by) `llvm::sys::fs::create_directories`

I think i forked it because it would not allow specifying permissions at the time, but now it does

need to test the clang version before. (Need 3.8)

Reply:
-> https://github.com/woboq/woboq_codebrowser/pull/71#discussion_r216251243